index.js

```diff
@@ -13,10 +13,10 @@ var Video = new Class({
 	},
 	volume: {
 		get: function() {
-			return (this.isYoutube) ? this.player.getVolume() : this.player.volume;
+			return (this.isYoutube) ? this.player.setVolume && this.player.getVolume() : this.player.volume;
```

Comment:
Shouldn't it be `this.player.getVolume` also maybe this shouldn't fail silently.